api/src/db/prisma.ts

```diff
@@ -18,6 +19,20 @@ const isTest = (workerId: string | undefined): workerId is string =>
   !!workerId && FREECODECAMP_NODE_ENV === 'development';
 
 const prismaPlugin: FastifyPluginAsync = fp(async (server, _options) => {
+  if (isTest(process.env.JEST_WORKER_ID)) {
+    // push the schema to the test db to setup indexes, unique constraints, etc
+    execSync('pnpm prisma db push', {
+      env: {
+        ...process.env,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
https://github.com/freeCodeCamp/freeCodeCamp/pull/51685, disabling naming convention is merged, this can be removed after rebase with main